api/src/pcapi/core/offerers/repository.py

```diff
@@ -581,3 +583,17 @@ def get_emails_by_offerer(offerer: models.Offerer) -> set[str]:
     }
     emails.discard(None)
     return emails
+
+
+def find_venues_of_offerer_from_siret(siret: str) -> Tuple[Offerer | None, list[models.Venue]]:
```

Comment:
(détail) Il me semble que tu peux utiliser `tuple` (le builtin) à la place de `typing.Tuple` : 

```suggestion
def find_venues_of_offerer_from_siret(siret: str) -> tuple[Offerer | None, list[models.Venue]]:
```